src/backend/parts/settings/Assistant.js

```diff
@@ -70,6 +71,26 @@ const Assistant = () => {
 			<Panel
 				header={ __( 'Assistant Settings', 'hyve-lite' ) }
 			>
+				<PanelRow>
+					<SelectControl
+						label={ __( 'Model', 'hyve-lite' ) }
+						help={ __( 'What model to use for Chat.', 'hyve-lite' ) }
+						options={ [
+							{
+								label: __( 'GPT 4o Mini', 'hyve-lite' ),
+								value: 'gpt-4o-mini'
+							},
+							{
+								label: __( 'GPT 3.5 Turbo 0125', 'hyve-lite' ),
+								value: 'gpt-3.5-turbo-0125'
```

Comment:
Not sure about this option. Even OpenAI does not recommend it. 

![image](https://github.com/user-attachments/assets/7aee9adb-8d4b-49d3-935b-233ed52f2ab1)

Reply:
I saw some still use gpt-3.5-turbo-0125 over 4o. We use 4o by default now.